### DISCUSSION TITLE
How do I disable the <C-x> binding in the picker?

### DISCUSSION THREAD

**Discussion by Hubro** [epwalsh/obsidian.nvim#650] [Q&A] [answered]
I want `<C-x>` to open a note in a split like it does in Telescope by default, but Obsidian overrides that mapping to `Create a new note from your query` instead. How do I restore the split behavior? I've tried:

```lua
picker = {
  mappings = {},
},
```

```lua
picker = {
  mappings = {
    new = nil,
  },
},
```

```lua
picker = {
  mappings = {
    new = "<C-F15>",
  },
},
```

None of these have any effect, not even the last one. `<C-x>` still creates a new note with the query content.

**Comment by epwalsh** [comment, marked as answer]
Hey @Hubro, try:

```lua
picker = {
  note_mappings = {
    new = "<C-F15>",
  },
},
```

**Reply by epwalsh** [reply]
I just realized the example in the README was wrong. Fixed here: 2cb8b3d

**Reply by Hubro** [reply]
```lua
    picker = {
      note_mappings = {
        new = nil,
      },
    },
```

Also works, which is nice since it avoids `<C-15>` in the help text in the picker title.
